core/src/commonMain/kotlin/com/powersync/db/PowerSyncDatabaseImpl.kt

```diff
@@ -85,7 +85,7 @@ internal class PowerSyncDatabaseImpl(
     }
 
     @OptIn(FlowPreview::class)
-    override suspend fun connect(connector: PowerSyncBackendConnector) {
+    override suspend fun connect(connector: PowerSyncBackendConnector, retryDelayMs: Long) {
```

Comment:
It looks like the Dart SDK has `crudThrottleTime` which is used as the upload delay and `retryDelay` which is used as the sync stream download delay. Maybe we should specify both of those here.

Reply:
Added that as a parameter as well.